testsuite/arquillian-utils/pom.xml

```diff
@@ -191,7 +191,7 @@
 
         <dependency>
             <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-nop</artifactId>
+            <artifactId>slf4j-simple</artifactId>
```

Comment:
I'm not sure why the test utils are pulling in `slf4j-nop` -- that just leads to no log output, making debugging difficult.